### DISCUSSION TITLE
sonarlint 内容翻译

### DISCUSSION THREAD

**Discussion by LinWanCen** [YiiGuxing/TranslationPlugin#935] [General]
sonarlint 插件的说明要是能调起翻译就好了

**Comment by YiiGuxing** [comment]
呃。。。。没有用过这个插件哦，你指的是哪一部分的翻译？发一些截图嘛，我好理解。

**Reply by LinWanCen** [reply]
![image](https://user-images.githubusercontent.com/14926671/150127891-935b206b-8c25-4751-a834-ad5c58fe1032.png)

**Reply by YiiGuxing** [reply]
如果你能选择你想要翻译的文本，可以在选择文本之后，可以尝试使用右键菜单上的翻译功能，如果这功能没有，则可以直接使用翻译快捷键进行翻译。
